concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1798,6 +1797,100 @@ jobs:
 
                   ../paas-cf/concourse/scripts/cf_push_on_git_change.sh graphite-nozzle
 
+        - task: create-paas-metrics-user
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-uaac
+                tag: 895cf6752c8ec64af05a3a735186b90acd3db65a
+            params:
+              ENABLE_METRICS: ${ENABLE_METRICS}
```

Comment:
This ins't going to parse. You may need to modify to fit what we're doing for datadog in the other comment.